Assets/VRTK/Scripts/Utilities/VRTK_PseudoSDKObject.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Pseudo SDK Object|Utilities|90063
+namespace VRTK
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// The GameObject that the Pseudo SDK Object script is applied to will become a child of the selected SDK Object.
+    /// </summary>
+    public class VRTK_PseudoSDKObject : MonoBehaviour
+    {
+        /// <summary>
+        /// Valid SDK Objects
+        /// </summary>
+        /// <param name="Boundary">The main camera rig/play area object that defines the player boundary.</param>
+        /// <param name="Headset">The main headset camera defines the player head.</param>
+        /// <param name="LeftController">The main object that defines the left controller.</param>
+        /// <param name="RightController">The main object that defines the right controller.</param>
+        public enum SDKObjects
+        {
+            Boundary,
+            Headset,
+            LeftController,
+            RightController
+        }
+
+        [Tooltip("The specific SDK Object to child this GameObject to.")]
+        public SDKObjects sdkObject = SDKObjects.Boundary;
+
+        protected VRTK_SDKManager sdkManager;
+
+        protected virtual void OnEnable()
+        {
+            VRTK_SDKManager sdkManager = VRTK_SDKManager.instance;
+            if (sdkManager != null)
+            {
+                sdkManager.LoadedSetupChanged += LoadedSetupChanged;
+            }
+            ChildToSDKObject();
+        }
+
+        protected virtual void OnDisable()
+        {
+            if (sdkManager != null)
+            {
+                sdkManager.LoadedSetupChanged -= LoadedSetupChanged;
+            }
+        }
+
+        protected virtual void LoadedSetupChanged(VRTK_SDKManager sender, VRTK_SDKManager.LoadedSetupChangeEventArgs e)
+        {
+            ChildToSDKObject();
+        }
+
+        protected virtual void ChildToSDKObject()
+        {
+            Vector3 currentPosition = transform.localPosition;
+            Quaternion currentRotation = transform.localRotation;
+            Vector3 currentScale = transform.localScale;
+
+            switch (sdkObject)
+            {
+                case SDKObjects.Boundary:
+                    transform.SetParent(VRTK_DeviceFinder.PlayAreaTransform());
```

Comment:
This may be cleaner by only setting a local `Transform parent` in the `switch` and then doing the `transform.SetParent(parent)` after the `switch`. (Just to save some typing and remove duplicated code.)